docker/llm/finetune/qlora/cpu/docker/Dockerfile

```diff
@@ -22,18 +46,25 @@ RUN apt-get update && \
     ln -s /usr/bin/python3.9 /usr/bin/python3 && \
     ln -s /usr/bin/python3 /usr/bin/python && \
     apt-get install -y python3-pip python3.9-dev python3-wheel python3.9-distutils && \
-    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+    curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
+# Install OpenSSH for MPI to communicate between containers
+    apt-get install -y --no-install-recommends openssh-client openssh-server && \
+    mkdir -p /var/run/sshd && \
+# Allow OpenSSH to talk to containers without asking for confirmation
+# by disabling StrictHostKeyChecking.
+# mpi-operator mounts the .ssh folder from a Secret. For that to work, we need
+# to disable UserKnownHostsFile to avoid write permissions.
+# Disabling StrictModes avoids directory and files read permission checks.
+    sed -i 's/[ #]\(.*StrictHostKeyChecking \).*/ \1no/g' /etc/ssh/ssh_config && \
+    echo "    UserKnownHostsFile /dev/null" >> /etc/ssh/ssh_config && \
+    sed -i 's/#\(StrictModes \).*/\1no/g' /etc/ssh/sshd_config
 
-# install torch and oneccl to reduce bigdl-llm size
-RUN pip3 install --upgrade pip && \
-    export PIP_DEFAULT_TIMEOUT=100 && \
-    pip install --upgrade torch==2.0.1 --index-url https://download.pytorch.org/whl/cpu && \
-    pip install oneccl_bind_pt -f https://developer.intel.com/ipex-whl-stable && \
-    # install CPU bigdl-llm
-    pip install --pre --upgrade bigdl-llm[all] -i https://pypi.tuna.tsinghua.edu.cn/simple/ && \
-    # install huggingface dependencies
-    pip install transformers==4.34.0 && \
-    pip install peft==0.5.0 datasets
+ADD https://raw.githubusercontent.com/intel-analytics/BigDL/main/python/llm/example/CPU/QLoRA-FineTuning/alpaca-qlora/alpaca_qlora_finetuning_cpu.py /bigdl/qlora_finetuning_cpu.py
```

Comment:
Hi @hzjane , I changed the Dockerfile to download alpaca_qlora_finetuning_cpu.py into image, as we discussed before. Moreover, I refactored files used to be under /, to /bigdl now